robot-service/src/main/webapp/js/properties.js

```diff
@@ -0,0 +1 @@
+parser.url = "";
```

Comment:
This one should have been added to our 'TODO' list. Ask Daniel ;) He started adding new functionality to out front-end, so maybe he is changing the concept now ;)